src/openMVG_Samples/siftPutativeMatches/CMakeLists.txt

```diff
@@ -7,6 +8,9 @@ TARGET_LINK_LIBRARIES(openMVG_sample_siftPutative
   openMVG_features
   openMVG_matching
   vlsift
-  stlplus)
+  PopSift::popsift
```

Comment:
the same here, popsift should be added automagically by `openMVG_features `, hence the find is not necessary.